src/interface/web/app/agents/page.tsx

```diff
@@ -318,41 +266,56 @@ export default function Agents() {
     }
 
     return (
-        <main className={styles.main}>
-            <h3
-                className='text-xl py-4'>
-                Agents
-            </h3>
+        <main className={`${styles.main} w-full ml-auto mr-auto`}>
+            <div className="float-right w-fit h-fit">
+                <NavMenu selected="Agents" />
+            </div>
             {
                 showLoginPrompt &&
                 <LoginPrompt
                     loginRedirectMessage="Sign in to start chatting with a specialized agent"
                     onOpenChange={setShowLoginPrompt} />
             }
-
-            <Alert>
-                <Info className="h-4 w-4" />
-                <AlertTitle>How this works</AlertTitle>
-                <AlertDescription>
-                    You can use any of these specialized agents to tailor to tune your conversation to your needs.
-                    {
-                        !authenticatedData &&
-                        <>
-                            <div className='mt-3' />
-                            <Button onClick={() => setShowLoginPrompt(true)}>
-                                <UserCircle className='w-4 h-4 mr-2' /> Sign In
-                            </Button>
-                        </>
-
-                    }
-                    <div className='mt-3' />
-                    <strong>Coming Soon:</strong> Support for making your own agents.
-                </AlertDescription>
-            </Alert>
-            <div className={styles.agentList}>
-                {data.map(agent => (
-                    <AgentCard key={agent.slug} data={agent} userProfile={authenticatedData} isMobileWidth={isMobileWidth} />
-                ))}
+            <div className={`${styles.pageLayout} w-full ml-auto mr-auto`}>
+                <div className={`${styles.sidePanel} top-0`}>
+                    <SidePanel
+                        webSocketConnected={true}
+                        conversationId={null}
+                        uploadedFiles={[]}
+                        isMobileWidth={isMobileWidth}
+                    />
+                </div>
+                <div className={`ml-auto mr-auto ${isMobileWidth ? "w-11/12" : "w-1/2"} pt-4`}>
```

Comment:
Can ml-auto and mr-auto be replace with just `mx-auto` (which should take care of x-axis margin)?
```suggestion
                <div className={`mx-auto ${isMobileWidth ? "w-11/12" : "w-1/2"} pt-4`}>
```